src/main/java/org/reaktivity/nukleus/tcp/internal/Counters.java

```diff
@@ -22,11 +22,13 @@
 {
     private final AtomicCounter routesSourced;
     private final AtomicCounter streamsSourced;
+    private final AtomicCounter streamsResetPartialWrite;
 
     Counters(CountersManager countersManager)
     {
         routesSourced = countersManager.newCounter("routesSourced");
         streamsSourced = countersManager.newCounter("streamsSourced");
+        streamsResetPartialWrite = countersManager.newCounter("streamsResetPartialWrites");
```

Comment:
I was thinking we should rename the existing counters just to `routes` and `streams` at some point, now that there are no independent counters for `routesTargeted` and `streamsTargeted`.
In the meantime, perhaps we can use `writesOverflowed` for the new counter name?
Later, we can always call it something like `overflows`.